docs/integrations_api.yml

```diff
@@ -141,80 +143,107 @@ paths:
   /devices/{id}:
     get:
       summary: Get a selected device
+      description: Returns the details of the selected devices, including its attributes.
       parameters:
         - name: id
           in: path
-          description: Device identifier
+          description: Device identifier.
           required: true
           type: string
       responses:
         200:
-          description: Device found.
+          description: Successful response - the device was found.
           schema:
             $ref: "#/definitions/Device"
+          examples:
+            application/json:
+              id: "291ae0e5956c69c2267489213df4459d19ed48a806603def19d417d004a4b67e"
+              attributes:
+                - name: "ip_addr"
+                  value: "1.2.3.4"
+                  description: "IP address"
+                - name: "mac_addr"
+                  value: "00.01:02:03:04:05"
+                  description: "MAC address"
+                - name: "ports"
+                  value:
+                    - "8080"
+                    - "8081"
+                  description: "Open ports"
+              updated_ts: "2016-10-03T16:58:51.639Z"
         404:
-          description: Device not found
+          description: The device was not found.
           schema:
             $ref: "#/definitions/Error"
         500:
-          description: Internal error
+          description: Internal server error.
           schema:
             $ref: "#/definitions/Error"
   /devices/{id}/group:
     get:
-      summary: Get a selected device's group.
+      summary: Get a selected device's group
       parameters:
         - name: id
           in: path
-          description: Device identifier
+          description: Device identifier.
           required: true
           type: string
       responses:
         200:
-          description: Success. In case device is not assigned to any group, group will be set as null.
+          description: |
+            Successful response.
+
+            If the device is not assigned to any group, the 'group' field will be set to 'null'.
           schema:
             $ref: "#/definitions/Group"
+          examples:
+            application/json:
+              group: "testing"
+        400:
+          description: Missing or malformed request params or body. See the error message for details.
         404:
-          description: Device not found
+          description: The device was not found.
           schema:
             $ref: "#/definitions/Error"
         500:
-          description: Internal error
+          description: Internal server error.
           schema:
             $ref: "#/definitions/Error"
     put:
-      summary: Add device to group.
+      summary: Add a device to a group
```

Comment:
Important property is that device can be only in one group. Adding it to another will move it to the new group.